<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pokedexClass</class>
 <widget class="QMainWindow" name="pokedexClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>pokedex</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCommandLinkButton" name="removeButton1">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>0</y>
      <width>25</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 0, 0, 150);</string>
    </property>
    <property name="text">
     <string>CommandLinkButton</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget1" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>275</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 0, 0);</string>
    </property>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>275</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
font: 12pt &quot;Jokerman&quot;;
color: rgb(255, 255, 255);
</string>
     </property>
     <widget class="QPushButton" name="bPokedex">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Pokedex</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bTeams">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>0</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Teams</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bOwners">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>0</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Owners</string>
      </property>
     </widget>
    </widget>
    <widget class="QListWidget" name="listPok">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>251</width>
       <height>521</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 13pt &quot;Jokerman&quot;;
background-color: rgb(170, 0, 0);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createTeam">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>560</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 13pt &quot;Jokerman&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Create team</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createOwner">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>560</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 13pt &quot;Jokerman&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Create owner</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget2" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>275</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 127);</string>
    </property>
    <widget class="QWidget" name="wikipok" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>241</width>
       <height>541</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 127);</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="namePokLayout"/>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>221</width>
        <height>1</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="imgPokLayout"/>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="imgType2Layout"/>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="imgType1Layout"/>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="stats1Layout"/>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="stats2Layout"/>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="stats3Layout"/>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>1</width>
        <height>91</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>221</width>
        <height>1</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="descriptionLayout" stretch="">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
      </layout>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>221</width>
        <height>1</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="abilitiesLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="heightLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_8">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>270</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="weightLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
      </layout>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>221</width>
        <height>1</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>221</width>
        <height>1</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>221</width>
        <height>1</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="eggGroupLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_10">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>300</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="speciesLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="maleLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_12">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>350</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="femaleLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="evolutionLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>221</width>
        <height>73</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="movesLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="prevPok">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>560</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 11pt &quot;Jokerman&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Previous</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextPok">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>560</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 11pt &quot;Jokerman&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget3" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>275</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 0, 0);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget4" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>0</y>
      <width>275</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 127);</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="removeButton2">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>0</y>
      <width>25</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 255, 150);
</string>
    </property>
    <property name="text">
     <string>CommandLinkButton</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="removeButton3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>875</x>
      <y>0</y>
      <width>25</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 255, 0, 150);</string>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="removeButton4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1175</x>
      <y>0</y>
      <width>25</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 150);</string>
    </property>
    <property name="text">
     <string>CommandLinkButton</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pokedex.qrc"/>
 </resources>
 <connections/>
</ui>
